lib/swagger/blocks.rb

```diff
@@ -33,30 +51,44 @@ def self.build_api_json(resource_name, swaggered_classes)
     end
 
     module InternalHelpers
+
       # Return [root_node, api_node_map] from all of the given swaggered_classes.
       def self.parse_swaggered_classes(swaggered_classes)
-        root_nodes = []
-        api_node_map = {}
-        models_nodes = []
+        root_nodes          = []
```

Comment:
Style note (guess I need a style guide! :) ): please just use single spaces around " = " instead of aligning assignments, here and elsewhere.